### DISCUSSION TITLE
How to filter the messages from lazy?

### DISCUSSION THREAD

**Discussion by chuan137** [LazyVim/LazyVim#168] [Q&A]
Hi guys,

I would like to hide the "Plugin Updates" messages on startup nvim, here is what I tried, and it did not work. What are the event and kind of such messages?

```
  {
    "folke/noice.nvim",
     routes = {
        -- don't show messages when buffer is written
        {
          filter = {
            event = "msg_show",
            kind = "",
            find = "written",
          },
          opts = { skip = true },
        },
        {
          filter = {
            event = "notify",
            kind = "",
            find = "Plugin Updates",
            -- has = true,
          },
          opts = { skip = true },
        },
      },
    },
  },
```

**Comment by dpetka2001** [comment]
You had a similar question in `noice` repo's discussions and you already got an answer from the developer 
for what you need to do to disable this kind of notifications. Didn't that solution work for you?

**Comment by chuan137** [comment]
my bad, I thought my old question was gone, forgot I put it over there
thanks for pointing it out

for anyone want the same, the answer is [here](https://github.com/folke/noice.nvim/discussions/330#discussioncomment-4786350)
